Tests/AWSSDKSwiftCoreTests/QueryEncoderTests.swift

```diff
@@ -128,6 +128,15 @@ class QueryEncoderTests: XCTestCase {
         testQuery(test, query:"A.1.entry=1&A.1.name=first")
     }
     
+    func testDataBlobEncode() {
+        struct Test : AWSShape {
+            static let _members = [AWSShapeMember(label: "A", required: true, type: .blob)]
+            let a : Data
+        }
+        let test = Test(a:Data("Testing".utf8))
+        testQuery(test, query:"A=Testing")
```

Comment:
If we replace with base64 encoding you will have to test against a URL percent encoded string. So your test would be
```
testQuery(test, query:"A=\(data.base64EncodedString().addingPercentEncoding(withAllowedCharacters: AWSClient.queryAllowedCharacters)!)")
```

Reply:
In QueryEncoderTests, we should not care params encode. In SES, that logic should guarantee by [urlEncodeQueryParams](https://github.com/swift-aws/aws-sdk-swift-core/blob/4.x.x/Sources/AWSSDKSwiftCore/AWSClient.swift#L450)

Reply:
You are right we shouldn't be doing the percent coding. Perhaps the test should be as follows given the function `testQuery` uses URLComponents to bulid the query string
```
        var components = URLComponents()
        components.queryItems = [URLQueryItem(name: "A", value: data.base64EncodedString())]
        testQuery(test, query: components.url!.query!)
```